### PR TITLE
Template DNSimple v0.7.11

### PR DESCRIPTION
**Pull request by janeczku**

### CODE REVIEW

File: infra-templates/dnsimple/4/rancher-compose.yml [hunk 1]
```diff
@@ -1,18 +1,13 @@
 # notemplating
 .catalog:
   name: "DNSimple DNS"
-  version: "v0.6.0-rancher2"
-  description: "Rancher External DNS service powered by DNSimple. Requires Rancher version v1.1.0"
-  minimum_rancher_version: v1.6.0-rc1
+  version: "v0.7.11"
+  description: "Rancher External DNS service powered by DNSimple"
+  minimum_rancher_version: v1.6.13-rc1
```
[the remark this answers is not in the document]
Comment (janeczku) [reply infra-templates/dnsimple/4/rancher-compose.yml]: The reason i put v1.6.13 as min version is that around that version there was change with regards to handling `# notemplating` tag in the compose files. I think @deniseschannon might know.

File: infra-templates/dnsimple/4/docker-compose.yml [hunk 2]
```diff
@@ -1,14 +1,16 @@
+# notemplating
 dnsimple:
-  image: rancher/external-dns:v0.6.0
+  image: rancher/external-dns:v0.7.11
   command: -provider=dnsimple
   expose:
    - 1000
   environment:
-    DNSIMPLE_TOKEN: ${DNSIMPLE_TOKEN}
-    DNSIMPLE_EMAIL: ${DNSIMPLE_EMAIL}
```
[the remark this answers is not in the document]
Comment (janeczku) [reply infra-templates/dnsimple/4/docker-compose.yml]: Correct, the provider no longer uses the email.

File: infra-templates/dnsimple/4/rancher-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (deniseschannon) [reply infra-templates/dnsimple/4/rancher-compose.yml]: reviewing the docs on when #notemplating worked correctly, it seems that we could set the min version as v1.6.0.

**Comment by janeczku** [comment]
@alena1108 PR updated per comments from you and @deniseschannon

**Comment by alena1108** [comment]
LGTM

**Pull request merged**